src/components/Navigator/NavigatorCardItem.vue

```diff
@@ -241,14 +241,14 @@ $depth-spacer-base-spacing: (
 $nesting-spacing: $card-horizontal-spacing + $card-horizontal-spacing-small;
 
 .navigator-card-item {
-  overflow: hidden;
   height: $item-height;
   display: flex;
   align-items: center;
 
-  @include on-keyboard-focus {
+  &:focus-within {
```

Comment:
This is a bit risky, as it got a wide spread adoption recently, but its OK on Safari based devices - https://caniuse.com/?search=focus-within

Reply:
I'm not very worried about its browser compatibility which it's similar to the one CSS Grid has.